src/Components/Actions/AddressAutocompleteAction/AddressAutocompleteAction.tsx

```diff
@@ -298,19 +314,16 @@ export const AddressAutocompleteAction: React.FC<AddressAutocompleteActionProps>
           onMouseLeave={() => setIsHovered(false)}
         >
           {props.tooltip ? <Tooltip tooltip={props.tooltip} /> : null}
-
-          <FakeInput
-            placeholder={props.placeholder}
-            value={confirmedAddressLine || searchTerm}
-            onClick={() => setIsAutocompleteActive(true)}
-            onFocus={() => setIsAutocompleteActive(true)}
-            size={
-              (confirmedAddressLine || searchTerm || props.placeholder).length
-            }
-          />
-          {confirmedPostalLine ? (
-            <PostalAddress>{confirmedPostalLine}</PostalAddress>
-          ) : null}
+          <Button onClick={handleButtonClick}>
+            <FakeInput
+              fakePlaceholder={confirmedAddressLine === '' || searchTerm === ''}
```

Comment:
`confirmedAddressLine` will never be an empty string... Shouldn't it be like:

```suggestion
              fakePlaceholder={confirmedAddressLine === undefined && searchTerm === ''}
```

Reply:
Ah yes, good catch, thank you! 🙏